src/modules/31-filestore/components/FileStoreContext/FileStoreContext.tsx

```diff
@@ -143,6 +145,19 @@ export const FileStoreContextProvider: React.FC<FileStoreContextProps> = (props:
     setGlobalSort(sortType)
   }
 
+  const { data: forceDeleteSettings } = useGetSettingValue({
+    identifier: SettingType.ENABLE_FORCE_DELETE,
+    queryParams: { accountIdentifier: queryParams.accountIdentifier },
+    lazy: false
+  })
+
+  React.useEffect(() => {
```

Comment:
Not a blocker, but `useEffect` is already imported by name, so no need for `React.`
```suggestion
  useEffect(() => {
```